lib/config.rb

```diff
@@ -57,12 +57,14 @@ def self.setting_files(config_root, env)
     [
       File.join(config_root, 'settings.yml').to_s,
       File.join(config_root, 'settings', "#{env}.yml").to_s,
-      File.join(config_root, 'environments', "#{env}.yml").to_s,
-
-      File.join(config_root, 'settings.local.yml').to_s,
-      File.join(config_root, 'settings', "#{env}.local.yml").to_s,
-      File.join(config_root, 'environments', "#{env}.local.yml").to_s
-    ].freeze
+      File.join(config_root, 'environments', "#{env}.yml").to_s
+    ].tap do |files|
+      files.concat([
+        File.join(config_root, 'settings.local.yml').to_s,
```

Comment:
It seems like this is the only file that should be affected by this. The next two lines would only include `settings/test.local.yml` and `environments/test.local.yml` which seem like they should still be included if somebody took the time to create them.

Reply:
Hmm, true. I don't quite see why somebody should do that, but if they add a file specifically for the test environment, they probably want it to be included. :grinning:

Reply:
Done.